rest/route/github.go

```diff
@@ -414,6 +417,21 @@ func (gh *githubHookApi) AddIntentForPR(pr *github.PullRequest, owner, calledBy
 	if err != nil {
 		return errors.Wrap(err, "creating GitHub patch intent")
 	}
+	// If there are no errors with the PR, verify that we aren't skipping CI before adding the intent (and send a message).
+	for _, label := range skipCILabels {
+		if strings.Contains(pr.GetTitle(), label) || strings.Contains(pr.GetBody(), label) {
```

Comment:
would it be worth making it case insensitive in case people write something like `[Skip-ci]`